src/intl/index.js

```diff
@@ -71,7 +73,9 @@ export default function reducer(state = buildInitialState(), action) {
       return state.updateIn(['languagePacks', locale, componentKey], iMap(), (nextState) => nextState.withMutations((map) => map
         .set('data', iMap(data))
         .set('isLoading', false)
-        .set('_loadedOnServer', !!global.BROWSER)
+        .set('_loadedOnServer', !global.BROWSER)
+        .delete('error')
```

Comment:
I think this should be covered by `LANGUAGE_PACK_REQUEST` as `LANGUAGE_PACK_SUCCESS` i think `LANGUAGE_PACK_SUCCESS` is proceeded by `LANGUAGE_PACK_REQUEST`